### DISCUSSION TITLE
Firebase list of services? I'm new to Firebase and many other things.

### DISCUSSION THREAD

**Discussion by myango1** [TheAlphamerc/flutter_twitter_clone#313] [General]
Firebase list of services? I'm new to Firebase and many other things.

Hi, can you list or point me to a file that list what exactly is used from Firebase, I'd like to see if these services or processes can be accomplished locally or with different providers or bypassed to review other aspects. I'm opening a firebase account but I'm totally new to this, and Flutter. I was able to replace many broken areas but the google plist seems to require an account.

**Comment by TheAlphamerc** [comment]
Hi @myango1, here is the steps to setup project and firebase services that we are using -> https://github.com/TheAlphamerc/flutter_twitter_clone/wiki/Gettings-Started

**Reply by myango1** [reply]
Appreciate it almost have it going, I set up Firebase and currently running the Authentication Demo for Flutter, works, now back to the TwitterClone. Almost there. I spent my life downloading and upgrading dependancies. lol. :)

**Comment by myango1** [comment]
this command fails from https://github.com/TheAlphamerc/flutter_twitter_clone/wiki/Gettings-Started
There's an extra command: 
keytool -exportcert -list -v \
-alias androiddebugkey -keystore ~/.android/debug.keystore
instead I used:
keytool -list -v -alias androiddebugkey -keystore ~/.android/debug.keystore

also the default password is "android", should also list for inexperienced users.

**Reply by TheAlphamerc** [reply]
Hi, here is the article about generating SHA-1 Keys -> https://developers.google.com/android/guides/client-auth

**Comment by myango1** [comment]
What's the reason for "you'll need to enable Google authentication." in https://github.com/TheAlphamerc/flutter_twitter_clone/wiki/Gettings-Started 
? 
I bypassed it for now. Hopefully I can work around it.

**Reply by TheAlphamerc** [reply]
There is google auth available in the app. If you don't want to use the feature then you can skip this.

**Comment by myango1** [comment]
The firebase console doesn't look like "Step 6 Add "FcmServerKey" to Firebase Remote Config Follow below steps to get FCM server key." The one I'm using sends you to IAM & Admin panel , to "service accounts". I have two entries there, so not sure. Under "Actions" there's a manage key options". "Cloud messaging" in Firebase Console Settings window has no "Add Server Key" button anymore. I'll skip android for now.
